cmd-dump.c

```diff
@@ -915,6 +920,19 @@ static void print_chrome_perf_event(struct uftrace_dump_ops *ops,
 				    struct uftrace_perf_reader *perf,
 				    struct uftrace_record *frs)
 {
+	uint64_t evt_id = frs->addr;
+
+	switch (evt_id) {
+		case EVENT_ID_PERF_COMM:
+			pr_out(",\n{\"ts\":0,\"ph\":\"M\",\"pid\":%d,"
+			       "\"name\":\"process_name\","
+			       "\"args\":{\"name\":\"%s\"}}",
+			       perf->tid, perf->u.comm.comm);
+			break;
+		case EVENT_ID_READ_PROC_STATM:
```

Comment:
This event should be handled in the `ops->task_event` callback.  Also please add the `default` case.